redwood-layout/src/androidTest/kotlin/app/cash/redwood/layout/ViewFlexContainerTest.kt

```diff
@@ -15,129 +15,115 @@
  */
 package app.cash.redwood.layout
 
-import android.annotation.SuppressLint
 import android.graphics.Color
 import android.graphics.drawable.ColorDrawable
 import android.widget.TextView
 import app.cash.paparazzi.DeviceConfig.Companion.PIXEL_6
 import app.cash.paparazzi.Paparazzi
+import app.cash.redwood.flexcontainer.AlignItems
+import app.cash.redwood.flexcontainer.FlexDirection
+import app.cash.redwood.flexcontainer.JustifyContent
 import org.junit.Rule
 import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.Parameterized
+
+@RunWith(Parameterized::class)
```

Comment:
I tend to use https://github.com/google/TestParameterInjector nowadays, but since the types and values here aren't actually enums it may not actually save any code like it normally should.

Reply:
Looks like it has the [same issue](https://github.com/google/TestParameterInjector/issues/22) with value classes as `Parameterized ` unfortunately. We'd still have to wrap all our params and do the cross product ourselves.